test/aliasTexting-test.js

```diff
@@ -0,0 +1,818 @@
+"use strict";
+
+var express = require("express");
+var Router = require("../lib/Router");
+var supertest = require("supertest");
+var nock = require("nock");
+var catapult = require("bandwidth");
+
+//nock.recorder.rec();
+var app = express();
+
+describe("TextApp", function () {
+	var router;
+	before(function () {
+		router = new Router();
+		app.use("/", router.router);
+	});
+	it("should serve the HTML page", function (done) {
+		supertest(app)
+			.get("/")
+			.expect("Content-Type", /html/)
+			.expect(200)
+			.end(done);
+	});
+	after(function () {
+		nock.cleanAll();
+	});
+	describe("should test the initialize function", function () {
+		before(function () {
+			nock("https://api.catapult.inetwork.com:443")
+				.get("/v1/users/u-37oyq5ser536gujhptoks6y/phoneNumbers?applicationId=a-hvszhcwftsdr6khcn4zl6my&numberState=ENABLED")
+				.reply(200, [ { "application" :"https://api.catapult.inetwork.com/v1/users/" +
+				"u-37oyq5ser536gujhptoks6y/applications/a-hvszhcwftsdr6khcn4zl6my",
+					"id" :"n-kins5yioy3ih4e7le4uuexa","number" :"+15123879534",
+					"nationalNumber" :"(512) 387-9534",
+					"name" :"[\"+18283290994\",null,\"Hey, it was nice meeting you tonight\"]",
+					"createdTime" :"2015-06-24T14:01:07Z","city" :"AUSTIN",
+					"state" :"TX","price" :"0.25","numberState" :"enabled" },
+					{ "application" :"https://api.catapult.inetwork.com/v1/users/" +
+					"u-37oyq5ser536gujhptoks6y/applications/a-hvszhcwftsdr6khcn4zl6my",
+					"id" :"n-kins5yioy3ih4e7le4uuexa","number" :"+19196513853",
+					"nationalNumber" :"(919) 651-3853",
+					"createdTime" :"2015-06-24T14:01:07Z","city" :"CARY",
+					"state" :"NC","price" :"0.25","numberState" :"enabled" },
+					{ "application" :"https://api.catapult.inetwork.com/v1/users/" +
+					"u-37oyq5ser536gujhptoks6y/applications/a-hvszhcwftsdr6khcn4zl6my",
+					"id" :"n-omcrk3jsuwwj4loovnzspyi","number" :"+15123877530",
+					"nationalNumber" :"(512) 387-7530",
+					"name" :"[\"+18283290994\",null,\"Hey, it was nice meeting you tonight\"]",
+					"createdTime" :"2015-06-24T13:58:42Z","city" :"AUSTIN",
+					"state" :"TX","price" :"0.25","numberState" :"enabled" } ],
+					{ "cache-control"   : "no-cache",
+					"content-type"      : "application/json",
+					date                : "Wed, 24 Jun 2015 18:41:27 GMT",
+					link                : "<https://api.catapult.inetwork.com/v1/users/" +
+					"u-37oyq5ser536gujhptoks6y/phoneNumbers?applicationId=a-hvszhcwftsdr6khcn4zl6my&page=0&size=25>; rel='first'",
+					server              : "Jetty(8.1.10.v20130312)",
+					"transfer-encoding" : "chunked",
+					connection          : "Close" });
+		});
+		it("should resolve successfully", function (done) {
+			router.initialize().then(function () {
+				done();
```

Comment:
Because there is no response from this function to verify, you should verify that your "nock" has been called. Take a look at https://github.com/pgte/nock#isdone for an example of how to do that.